templates/includes/sidebar.html

```diff
@@ -1,4 +1,9 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <p>User: {{ user.first_name }}</p>
```

Comment:
The username is displayed correctly when the user is authenticated. However, ensure that clicking on the username leads to the corresponding Driver detail page as per the task requirements.